messybrainz/cache.py

```diff
@@ -188,6 +188,6 @@ def _prep_list(l, namespace=None):
 
 def _prep_dict(dictionary, namespace=None):
     """Wrapper for _prep_key function that works with dictionaries."""
-    for key in dictionary.keys():
+    for key in list(dictionary.keys()):
```

Comment:
I don't think that this `list()` call is needed - we're just using `dictionary.keys()` as an iterator here, we don't need to index into it.

Having said that, @mayhem and I already talked about moving the cache to redis for MsB, so I'm not too worried about leaving this and removing it next.